Sources/LLVM/ArrayType.swift

```diff
@@ -31,6 +31,18 @@ public struct ArrayType: IRType {
     }
   }
 
+  /// Creates a constant array value of i8 from a swift string.
+  ///
+  /// - parameter string: A string to create a null terminated array from.
```

Comment:
Can you be explicit about this being the UTF-8 bytes of the string?

Reply:
> Creates a constant, null terminated array value of utf8 bytes from string's utf8 member.
Good?

Reply:
Use “UTF-8” for the first one and surround the 2nd `utf8` in code voice and we’re good.